sdk/Integration/flask_client/identity_auth.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+from flask import request as flask_req
+from flask import session, redirect
+
+from sdk.Integration.flask_client.framework import FlaskFramework
+from sdk.constants.common import TOKEN_RESPONSE, REDIRECT
+from sdk.constants.token import ACCESS_TOKEN, ID_TOKEN, AUTHORIZATION_CODE, \
+    STATE
+from sdk.exception.identityautherror import IdentityAuthError
+from sdk.identity_auth import IdentityAuth
+
+logger = logging.getLogger(__name__)
+
+
+class FlaskIdentityAuth(IdentityAuth):
```

Comment:
```suggestion
class FlaskIdentityAuth(IdentityAuth):
```

Shall we change the class to `AsgardeoAuth` ?

Reply:
Sure